protoutil/txutils_test.go

```diff
@@ -230,7 +230,7 @@ func TestCreateSignedTx(t *testing.T) {
 		},
 	}}
 	_, err = protoutil.CreateSignedTx(prop, signID, responses...)
-	require.Error(t, err, "Expected error with no endorsements")
```

Comment:
I don't understand how this error change is related to protobuf or etcd changes

Reply:
fixed